cmd/siad/server.go

```diff
@@ -79,8 +79,8 @@ type (
 		RootTarget types.Target `json:"roottarget"`
 		RootDepth  types.Target `json:"rootdepth"`
 
-		MaxAdjustmentUp   *big.Rat `json:"maxadjustmentup"`
-		MaxAdjustmentDown *big.Rat `json:"maxadjustmentdown"`
+		MaxTargetAdjustmentUp   *big.Rat `json:"maxtargetadjustmentup"`
+		MaxTargetAdjustmentDown *big.Rat `json:"maxtargetadjustmentdown"`
```

Comment:
Can't remove these entirely, need to keep the deprecated values `MaxAdjustmentUp` and `MaxAdjustmentDown`  that are identical to the newly named values, so that old programs don't experience a break in compatibility.

Reply:
@DavidVorick you mean just keep the json properties the same as before? i.e MaxTargetAdjustmentUp will use `json:"maxadjustmentup"` and MaxTargetAdjustmentDown will use `json:"maxadjustmentdown"`

Reply:
```
	// SiaConstants is a struct listing all of the constants in use.
	SiaConstants struct {
		BlockFrequency         types.BlockHeight `json:"blockfrequency"`
		BlockSizeLimit         uint64            `json:"blocksizelimit"`
		ExtremeFutureThreshold types.Timestamp   `json:"extremefuturethreshold"`
		FutureThreshold        types.Timestamp   `json:"futurethreshold"`
		GenesisTimestamp       types.Timestamp   `json:"genesistimestamp"`
		MaturityDelay          types.BlockHeight `json:"maturitydelay"`
		MedianTimestampWindow  uint64            `json:"mediantimestampwindow"`
		SiafundCount           types.Currency    `json:"siafundcount"`
		SiafundPortion         *big.Rat          `json:"siafundportion"`
		TargetWindow           types.BlockHeight `json:"targetwindow"`

		InitialCoinbase uint64 `json:"initialcoinbase"`
		MinimumCoinbase uint64 `json:"minimumcoinbase"`

		RootTarget types.Target `json:"roottarget"`
		RootDepth  types.Target `json:"rootdepth"`

		MaxAdjustmentUp   *big.Rat `json:"maxadjustmentup"`
		MaxAdjustmentDown *big.Rat `json:"maxadjustmentdown"`
		MaxTargetAdjustmentUp   *big.Rat `json:"maxtargetadjustmentup"`
		MaxTargetAdjustmentDown *big.Rat `json:"maxtargetadjustmentdown"`

		SiacoinPrecision types.Currency `json:"siacoinprecision"`
}
```

Change the struct to this, that way the new fields and old fields are both supported. We have to use duplication when we change a field name to preserve compatibility and also provide the fixed endpoint. We do this in a few other places in the API too.